Connected web3 provider at 10.195.2.38:8002
Date.now(): 1579248753907 startTime: 1579248862873 delay: 108969
Deploying smart contract from 0x6fff54aa0710e9b47f5999f129f3ed301ea58d5a
TX HASH:
 0x83b22038aae97f0d4536837b10053f39c83aab1fd218642cc4f8d209633e5619
RECEIPT:
 {
  blockHash: '0x55a2bf332ec8e17e5de9bec9dd03614403dff20fb8569119b0b89ee3a7c2b0ab',
  blockNumber: 61,
  contractAddress: '0x8f6983a2F40df2F1d456A440a6f003AF231B7ef7',
  cumulativeGasUsed: 218495,
  from: '0x6fff54aa0710e9b47f5999f129f3ed301ea58d5a',
  gasUsed: 218495,
  logs: [],
  logsBloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
  status: true,
  to: null,
  transactionHash: '0x83b22038aae97f0d4536837b10053f39c83aab1fd218642cc4f8d209633e5619',
  transactionIndex: 0
}
Contract at: 0x8f6983a2F40df2F1d456A440a6f003AF231B7ef7

Evaluating  4500txs...

Sending TXS started at 1579248862956 ...
startingBlock: 170
... Sending TXS finished at 1579248962899
Total send duration: 99943
... Waiting for mining process to finish! ...
(node:22380) UnhandledPromiseRejectionWarning: Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at processTimers (internal/timers.js:472:9)
(node:22380) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:22380) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
ERROR:
 Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at processTimers (internal/timers.js:472:9)
========================================================
========================================================

/home/ubuntu/benchprezz/benchmark/tpsUnsigned.js:113
                throw 'Blockchain capabilities EXCEEDED - ABORTING test run!';
                ^
Blockchain capabilities EXCEEDED - ABORTING test run!
